recipes/open-simulation-interface/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import re
+import sys
+import shutil
+
+required_conan_version = ">=1.33.0"
+
+class OpenSimulationInterfaceConan(ConanFile):
+    name = "open-simulation-interface"
+    homepage = "https://github.com/OpenSimulationInterface/open-simulation-interface"
+    description = 'Generic interface environmental perception of automated driving functions in virtual scenarios'
+    topics = ("asam", "adas", "open simulation", "automated driving")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MPL 2.0"
```

Comment:
```suggestion
    license = "MPL-2.0"
```

https://spdx.org/licenses/MPL-2.0.html